core/src/main/java/org/hisp/dhis/android/sdk/controllers/tracker/TrackerDataSender.java

```diff
@@ -350,8 +342,6 @@ static void sendEnrollmentChanges(DhisApi dhisApi, boolean sendEvents) throws AP
         }
         if (enrollments.size() <= 1) {
```

Comment:
I don´t understand because this condition is here and I have found a bug related to this condition. 
I have created a TEI and Enrollment, cancel enrollment then create new enrollment, when automatic sync is fired, this condition avoids send existed two enrollments but parent method after this sends events and a conflict is responded from the server because enrollment has not been sending previously.
https://drive.google.com/file/d/0B7P8M8iuIGQvcVdhQk5zYmlWc0U/view?usp=sharing